### DISCUSSION TITLE
Text picture different layers compose a picture

### DISCUSSION THREAD

**Discussion by a123likeyou** [ImageMagick/ImageMagick#2897] [Help]
![aaa](https://user-images.githubusercontent.com/22471187/100696024-835aa480-33cd-11eb-9197-a484d930d73f.png)
It belongs to three layers: background layer, picture layer and text layer. How to realize this picture with one command？Thank you for your help

**Comment by snibgo** [comment]
If you have three input images, you could use a command like this:
```
magick background.png picture.png text.png -layers merge out.png
```

**Reply by a123likeyou** [reply]
Thank you, but this text layer is font type, not a picture similar to
 -font f -undercolor red  -pointsize 50 -weight ExtraBold -style Italic  -draw "fill blue text 40,100 'Copyright'" 
This command is stitched together with the picture

**Reply by a123likeyou** [reply]
> 如果您有三个输入图像，则可以使用如下命令：
> 
> ```
> magick background.png picture.png text.png -layers merge out.png
> ```

Thank you, but this text layer is font type, not a picture similar to
-font f -undercolor red -pointsize 50 -weight ExtraBold -style Italic -draw "fill blue text 40,100 'Copyright'"
This command is stitched together with the picture
Text layer is not a picture, we need to join together
